modules/nf-core/survivor/stats/main.nf

```diff
@@ -23,11 +23,17 @@ process SURVIVOR_STATS {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+    def is_compressed = vcf.getName().endsWith(".gz") ? true : false
+    vcf_name = vcf.getName().replace(".gz", "")
 
     """
+    if [ "$is_compressed" == "true" ]; then
+        gzip -c -d $vcf > $vcf_name
```

Comment:
Can you also remove this file again after the tool ran? (gotta think of those work directories :grin:)